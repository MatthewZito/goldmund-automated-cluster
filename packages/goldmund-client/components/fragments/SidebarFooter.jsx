import React from "react";

const SidebarFooter = () => {
    return (
        <nav className="nav-footer">
            <p className="nav-footer-social-buttons">
                <a aria-label="instagram" rel="noreferrer" className="fa-icon" target="_blank" href="https://www.instagram.com/magister_zito" title="instagram link">
                    <i className="fab fa-instagram" aria-hidden="true"></i>
                </a>
                <a aria-label="youtube" rel="noreferrer" className="fa-icon" target="_blank" href="https://www.youtube.com/user/fasolplanetarium/videos" title="youtube link">
                    <i className="fab fa-youtube" aria-hidden="true"></i>
                </a>
                <a aria-label="github" rel="noreferrer" className="fa-icon" target="_blank" href="https://github.com/MatthewZito" title="github link">
                    <i className="fab fa-github" aria-hidden="true"></i>
                </a>
            </p>
            <p>Â© goldmund</p>
        </nav>  
    );
}
export default SidebarFooter;
    