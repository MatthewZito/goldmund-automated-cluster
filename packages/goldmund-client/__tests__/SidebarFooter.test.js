import { shallow } from "enzyme";
import React from 'react';
import SidebarFooter from "../components/fragments/SidebarFooter.jsx";

it("should render the SidebarFooter component", () => {
    expect(shallow(<SidebarFooter />).debug()).toMatchSnapshot();
});