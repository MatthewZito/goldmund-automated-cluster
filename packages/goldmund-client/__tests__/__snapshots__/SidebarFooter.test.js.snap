// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the SidebarFooter component 1`] = `
"<nav className=\\"nav-footer\\">
  <p className=\\"nav-footer-social-buttons\\">
    <a className=\\"fa-icon\\" target=\\"_blank\\" href=\\"https://www.instagram.com/magister_zito\\" title=\\"\\">
      <i className=\\"fab fa-instagram\\" aria-hidden=\\"true\\" />
    </a>
    <a className=\\"fa-icon\\" target=\\"_blank\\" href=\\"https://www.youtube.com/channel/UCg3SBNPF1QyMeL-GtrKkvRQ/about?view_as=public\\" title=\\"\\">
      <i className=\\"fab fa-youtube\\" aria-hidden=\\"true\\" />
    </a>
    <a className=\\"fa-icon\\" target=\\"_blank\\" href=\\"https://github.com/MatthewZito\\" title=\\"\\">
      <i className=\\"fab fa-github\\" aria-hidden=\\"true\\" />
    </a>
  </p>
  <p>
    Â© goldmund
  </p>
</nav>"
`;
